src/download.rs

```diff
@@ -248,19 +248,16 @@ pub fn download_file(
             copy_largebuf(&mut pb.wrap_read(response), &mut buffer)?;
             let downloaded_length = pb.position() - starting_length;
             pb.finish_and_clear();
-            let time_taken = starting_time.elapsed().as_secs();
-            if let Some(speed) = downloaded_length.checked_div(time_taken) {
+            let time_taken = starting_time.elapsed();
+            if time_taken != Duration::from_nanos(0) {
                 eprintln!(
-                    "Done. {} in {} ({}/s)",
+                    "Done. {} in {:?} ({}/s)",
                     HumanBytes(downloaded_length),
-                    humantime::format_duration(Duration::from_secs(time_taken)),
-                    HumanBytes(speed)
+                    Duration::from_micros(time_taken.as_micros() as u64),
```

Comment:
Can we keep using `humantime`? the maximum unit `Duration` supports is seconds while `humantime` can display time in minutes, hours, days, and so on.

Reply:
Fixed.